.circleci/config.yml

```diff
@@ -51,6 +51,7 @@ commands:
       path:
         type: string
     steps:
+      - run: echo "export const version = '`git describe --always --dirty`';" > ./srv/version.ts
```

Comment:
I think there's a typo: `./srv/...`
I don't really get the interest of removing the version from the package.json to add it in the src folder, why do you want to do it this way?

Reply:
You were not marked as reviewer anymore while I was testing test ;-)

Reply:
There is no way to get the package.json version in live code in a clean, safe way (without exporting everything for the package.json) and compatible with a decent release process.

Reply:
And as I added in the PR description, so that we "Make sure dev deployments have an automated identifiable version number so that sentry bug report does not become a mess"

Reply:
I found a way to inject env vars with webpack `DefinePlugin` ([article[(https://medium.com/@fidelisclayton/system-environment-variables-in-angular-1f4a922c7b4c)), it might be cleaner but it's too overkill in my opinion, your call.

Reply:
It is a quite old post, it is not working as is, so I'll give up this option for now, even it looks quite a good idea.